spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -368,6 +372,30 @@ def perform_request
     end
   end
 
+  describe 'PATCH course' do
+    let(:path) do
+      "/api/v2/providers/#{provider.provider_code}" +
+        "/courses/#{course.course_code}"
+    end
+    let(:accrediting_provider) { create :provider }
+    let(:course) { create(:course, provider: provider, site_statuses: [site_status]) }
+    let(:site_status) { build(:site_status, :new) }
+
+    before do
+      course
+    end
+
+    it 'can update the accrediting provider' do
+      patch path, headers: { 'HTTP_AUTHORIZATION' => credentials }, params: { 'course' => { 'accrediting_provider_code' => accrediting_provider.provider_code } }
+      expect(JSON.parse(response.body)['data']['attributes']['accrediting_provider_code']).to eq(accrediting_provider.provider_code)
+      expect(provider.accredited_bodies).to eq([{
```

Comment:
wouldn't it be more direct to test `course.accredited_body`, it seems like a side-effect that `provider.accredited_bodies` is affected

I think testing `provider.accredited_bodies` makes this test more fragile to changes in the unrelated roll-up of course info into `provider.accredited_bodies`